extensions/deepcast/package.json

```diff
@@ -690,6 +690,36 @@
           "value": "paste"
         }
       ]
+    },
+    {
```

Comment:
It would be nice to have the transliteration feature added in a separate PR since this is about adding formality.

Reply:
I think that since its such a small change, its unnecessary to make a separate PR